SPECS/containerized-data-importer/cdi_containers_meta.service

```diff
@@ -0,0 +1,5 @@
+<service name="cdi_containers_meta">
```

Comment:
I don't think we need this. The containers corresponding to these rpms will be build using the golden container workflow. Please remove.